packages/web-twig/src/Resources/components/Field/README.md

```diff
@@ -0,0 +1,46 @@
+# ValidationText
```

Comment:
Not sure about this main heading. Maybe `Field` will be better based on the field structure and mention `ValidationText` as subcomponents.